scopes/harmony/cli/help.ts

```diff
@@ -19,8 +19,8 @@ type GroupContent = {
   description: string;
 };
 
-export function formatHelp(commands: CommandList, groups: GroupsType, docsDomain: string) {
-  const helpProps = groupCommands(commands, groups);
+export function formatHelp(commands: CommandList, groups: GroupsType, docsDomain: string, showPrivateCommands = false) {
```

Comment:
why do you use two terms? I guess we could use either private or internal for all variable/option names

Reply:
I actually started with `--private`, but then, the variable needs to be `private` as well, and that's impossible because `private` is a preserve word.

Reply:
I can do `--include-private` maybe. but it's a bit long